community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/main.tf

```diff
@@ -299,9 +307,16 @@ locals {
     filename = "z_setup_external.sh"
     content  = data.local_file.setup_external.content
   }]
+  chs_gpu_health_check = [{
+    filename = "chs_gpu_health_check.sh"
```

Comment:
I think filename should be either `Z` or `A` prefixed to ensure that it runs at some predictable time